tests/foreman/ui/test_remoteexecution.py

```diff
@@ -404,6 +428,12 @@ def test_positive_run_default_job_template(self):
                     job_template='Run Command - SSH Default',
                     options_list=[{'name': 'command', 'value': 'ls'}]
                 )
+                # get job invocation id from the current url
```

Comment:
@pondrejk could you expose the results via the `AssertError`?
The logging is not necessarily turned `on` or set to `debug`